simulations/vip-398/arbitrumone.ts

```diff
@@ -0,0 +1,294 @@
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [PRIME_ABI], ["MarketAdded"], [4]);
+      await expectEvents(txResponse, [PRIME_ABI], ["MintLimitsUpdated"], [1]);
+      await expectEvents(txResponse, [PROTOCOL_SHARE_RESERVE_ABI], ["DistributionConfigAdded"], [6]);
+      await expectEvents(txResponse, [PROTOCOL_SHARE_RESERVE_ABI], ["DistributionConfigUpdated"], [2]);
+    },
+  });
+
+  describe("Post-VIP behavior", () => {
+    let prime: Contract;
+    let plp: Contract;
+
+    before(async () => {
+      prime = new ethers.Contract(ARBITRUM_PRIME, PRIME_ABI, provider);
+      plp = new ethers.Contract(ARBITRUM_PLP, PLP_ABI, provider);
+    });
+
+    it("prime should have correct pool registry address", async () => {
+      expect(await prime.poolRegistry()).to.be.equal(arbitrumone.POOL_REGISTRY);
+    });
+
+    it("Comptroller lst and core should have correct Prime token address", async () => {
+      const comptrollerCore = new ethers.Contract(ARBITRUM_COMPTROLLER_CORE, COMPTROLLER_ABI, provider);
+      expect(await comptrollerCore.prime()).to.be.equal(ARBITRUM_PRIME);
+
+      const comptrollerLst = new ethers.Contract(ARBITRUM_COMPTROLLER_LST, COMPTROLLER_ABI, provider);
+      expect(await comptrollerLst.prime()).to.be.equal(ARBITRUM_PRIME);
+    });
+
+    it("Prime should contain correct markets", async () => {
+      expect((await prime.markets(ARBITRUM_VUSDC_CORE))[4]).to.be.equal(true);
+      expect((await prime.markets(ARBITRUM_VUSDT_CORE))[4]).to.be.equal(true);
+      expect((await prime.markets(ARBITRUM_VWBTC_CORE))[4]).to.be.equal(true);
+      expect((await prime.markets(ARBITRUM_VWETH_LST))[4]).to.be.equal(true);
+    });
+
+    it("Prime should have correct number of revocable and irrevocable tokens", async () => {
+      expect(await prime.irrevocableLimit()).to.equal(0);
+      expect(await prime.revocableLimit()).to.equal(500);
+    });
+
+    it("Plp should have correct tokens", async () => {
+      expect(await plp.lastAccruedBlockOrSecond(ARBITRUM_USDT)).to.be.gt(0);
+      expect(await plp.lastAccruedBlockOrSecond(ARBITRUM_USDC)).to.be.gt(0);
+      expect(await plp.lastAccruedBlockOrSecond(ARBITRUM_WETH)).to.be.gt(0);
+      expect(await plp.lastAccruedBlockOrSecond(ARBITRUM_WBTC)).to.be.gt(0);
+    });
+  });
+
+  describe("Owner checks", () => {
+    it("Timelock should be the owner of all converters", async () => {
+      for (const converter of converters) {
+        const Converter = new ethers.Contract(converter, SINGLE_TOKEN_CONVERTER_ABI, provider);
+        expect(await Converter.owner()).to.equal(arbitrumone.GUARDIAN);
+      }
+    });
+
+    it("Timelock should be the owner of ConverterNetwork", async () => {
+      expect(await converterNetwork.owner()).to.equal(arbitrumone.GUARDIAN);
+    });
+
+    it("Timelock should be the owner of XVSVaultTreasury", async () => {
+      expect(await xvsVaultTreasury.owner()).to.equal(arbitrumone.GUARDIAN);
+    });
```

Comment:
I would replace "Timelock" with "Guardian" in the descriptions, to be precise

Reply:
fixed, https://github.com/VenusProtocol/vips/pull/429/commits/563c701d661ec184a187dd111304ae60c8749e57